tools/system_libs.py

```diff
@@ -909,16 +920,7 @@ def check_tag():
     def unpack():
       logger.warning('unpacking port: ' + name)
       shared.safe_ensure_dirs(fullname)
-      if is_tarbz2:
-        z = tarfile.open(fullpath, 'r:bz2')
-      else:
-        z = zipfile.ZipFile(fullpath, 'r')
-      try:
-        cwd = os.getcwd()
-        os.chdir(fullname)
-        z.extractall()
-      finally:
-        os.chdir(cwd)
+      shutil.unpack_archive(fullpath, fullname)
```

Comment:
Again python3 only I think.

Perhaps add a TODO here that when we switch to python3 we should convert this.

Reply:
Done. Thanks!